AWS/debian/debian-8.json

```diff
@@ -60,7 +60,7 @@
     {
       "type": "shell",
       "inline": [
-        "sudo  rm -rf /etc/apt/apt.conf.d/50unattended-upgrades",
+        "sudo  rm -rf /etc/apt/apt.conf.d/20unattended-upgrades",
```

Comment:
should be 20auto-upgrades, not 20 unattended-upgrades

Reply:
even safer would be to 
```bash
echo "" > /etc/apt/apt.conf.d/20auto-upgrades
```
so any package updates don't re-create the file.

Reply:
I guess since the salt-jenkins states make sure it's absent, it shouldn't matter and we can just delete it.

Reply:
Fixed